lambda_functions/v1/functions/lpa_codes/app/api/code_generator.py

```diff
@@ -73,9 +73,6 @@ def get_codes(database, key=None, code=None):
 
     table = database.Table(lpa_codes_table())
 
-    return_fields = (
-        "lpa, actor, code, active, last_updated_date, dob, expiry_date, generated_date"
```

Comment:
We should keep the projection, it ensures we get back what we need and not what we don't. I'd ideally like to see a projection passed in so the query returns only what the handler requires